server/src/entities/Queue.ts

```diff
@@ -13,7 +13,8 @@ export class Queue extends BaseEntity {
   @Column()
   position: number
 
-  @ManyToOne(type => Song, song => song.id)
+  @ManyToOne(type => Song, song => song.id, {eager:true})
```

Comment:
I think once we get through our initial load testing, we may want to try and optimize this with the work around mentioned in our last class. (To avoid using eager:true whenever possible). 

No need to mess with this now-- and really it might be fine later. I just wanted to put this here to make a note for later if needed. Just mark this comment as read and we can go ahead with the merge.  :)